content/docs/get-started/serving.md

```diff
@@ -31,10 +34,37 @@ That's it! You can check out the other available server implementations in
 
 ## Making requests
 
-While serving the model, you can open Swagger UI (OpenAPI) at
-[http://localhost:8080/docs](http://localhost:8080/docs) to check out OpenAPI
-spec and query examples.
+While the model server is running, you can use your browser to open the Swagger
+UI (OpenAPI) at [http://localhost:8080/docs](http://localhost:8080/docs) to
+check out OpenAPI spec and query examples.
+
+Or, alternatively, launch requests at the server from a different terminal,
+Using `curl` or `httpie`. For example:
 
-MLEM can help you with not only serving, but also querying the model. Using MLEM
-API or CLI you can send requests to your model without implementing the "client"
-part. Learn more about this in [User Guide](/doc/user-guide/serving/).
+```cli
+$ curl -X 'POST' \
+  'http://0.0.0.0:8080/predict_proba' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "data": {
+    "values": [
+      {
+        "sepal length (cm)": 0,
+        "sepal width (cm)": 1,
+        "petal length (cm)": 2,
+        "petal width (cm)": 3
+      }
+    ]
+  }
+}'
+[[0.47,0.24,0.29]]
+```
```

Comment:
Similar example was in the first version of GS. For the sake of brevity, I removed it in the last version. I understand the reason: having a usage example is right. I did the same in `get-started/building` (`docker run...`). But it makes page big. I can suggest hiding it in `<details></details>` block to achieve both goals.

@jorgeorpinel WDYT?

Reply:
@aguschin  -> moved to details blcok